README.md

```diff
@@ -310,6 +311,27 @@ In darwin, `libclang.dylib`, In Linux, `libclang.so`.
 C language standard version option.  
 If not set, deoplete-go use `c11`(latest) version.
 
+### `g:deoplete#sources#go#goos`
+#### Set GOOS environment variable when calling `gocode`
+
+| **Default**  | `''`      |
+|--------------|-----------|
+| **Required** | No        |
+| **Type**     | string    |
+| **Example**  | `windows` |
+
```

Comment:
> It would be great if add available variable list, like g:deoplete#sources#go#sort_class.

Do you mean as an inclusive list of GOOS values?

> Please also add auto variable.

Add it as the example?